cime_config/namelist_definition_drv.xml

```diff
@@ -1694,6 +1694,18 @@
     </values>
   </entry>
 
+  <entry id="num_icesheets">
```

Comment:
I'm wondering if this needs to be here - it seems like it may be better to query the cism model to tell you the number of ice sheets?